assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js

```diff
@@ -16,6 +16,63 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import Tab from '@material/react-tab';
+import TabBar from '@material/react-tab-bar';
```

Comment:
Let's put external dependencies to be above wp dependencies:

```suggestion
/**
 * External dependencies
 */
import Tab from '@material/react-tab';
import TabBar from '@material/react-tab-bar';

/**
 * WordPress dependencies
 */
import { useCallback } from '@wordpress/element';
import { __ } from '@wordpress/i18n';
```